frontend/src/metabase/palette/components/PaletteResultsList.tsx

```diff
@@ -120,7 +120,7 @@ export const PaletteResultList: React.FC<PaletteResultListProps> = props => {
       if (item.command) {
         item.command.perform(item);
         query.toggle();
-      } else {
+      } else if (!item.extra?.href) {
         query.setSearch("");
         query.setCurrentRootAction(item.id);
       }
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +128]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->With this change, we don't have to include a no-op `perform` function when adding an external link to the command palette results.<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/metabase/metabase/47862?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->